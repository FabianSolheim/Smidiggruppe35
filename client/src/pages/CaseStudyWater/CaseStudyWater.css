body{
    margin: 0;
}

.header {
    width: 100%;
    height: 600px;
    background-image: url("../images/waterBackground.svg");
    background-repeat: no-repeat;
    background-size: cover;
    background-position: center;
    text-align: center;
    font-size: 64px;
}
.top {
    font-size: 0px;
}
.headline {
    margin-top: 210px;
    font-size: 64px;
    font-family: "Montserrat", sans-serif;
}
.description {
    font-size: 30px;
    font-family: "Montserrat", sans-serif;
}
.description em {
    color: #034F7A;
    font-style: normal;
}
.consultationBtn {
    margin-top: 80px;
    font-family: "Montserrat", sans-serif;
    border-radius: 0;
    height: 50px;
    width: 200px;
    background-color: #034F7A;
    color: white;
    border: 0;
    font-size: 16px;
}


@media only screen and (max-width: 750px) {


    .headline {
        font-size: 34px;
        margin-top: 100px;
    }
    .description {
        font-size: 16px;
    }
    .consultationBtn {
        font-size: 16px;
        margin-top: 60px;
    }
    .header {
        max-height: 5em;
    }

}