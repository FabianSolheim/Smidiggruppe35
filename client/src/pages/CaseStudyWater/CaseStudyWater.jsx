import React from "react";
import "./CaseStudyWater.css";

export function CaseStudyWater() {
  return (
    <>
      <div className="header">
        <div>
          <p className="top">t</p>
          <p className="headline">
            <strong>Water</strong>
          </p>
          <p className="description">
            <strong>
              <em>785 million</em>
            </strong>{" "}
            people in the world live{" "}
            <strong>
              <em>without</em>
            </strong>
            <br /> clean water.
          </p>
        </div>
        <button className="consultationBtn">
          <strong>Book a Consultation</strong>
        </button>
      </div>
    </>
  );
}
