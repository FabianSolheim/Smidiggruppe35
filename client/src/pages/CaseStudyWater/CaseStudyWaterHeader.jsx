import React from "react";
import "./CaseStudyWater.css";
import { HeaderBox } from "../../components/HeaderBox/HeaderBox";

export function CaseStudyWaterHeader() {
  return (
    <>
      <div className="header">
        {HeaderBox(
          "../../images/waterBackground.svg",
          "Water",
          "black",
          <p>
            <strong>
              <em>785 million</em>
            </strong>{" "}
            people in the world live{" "}
            <strong>
              <em>without</em>
            </strong>
            <br /> clean water.
          </p>,
          "black"
        )}
        <button className="consultationButton">
          <strong>Book a Consultation</strong>
        </button>
      </div>
    </>
  );
}
