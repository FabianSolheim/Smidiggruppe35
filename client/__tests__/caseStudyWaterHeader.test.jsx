import React from "react";
import ReactDOM from "react-dom";
import { MemoryRouter } from "react-router-dom";
import { CaseStudyWaterHeader } from "../src/pages/CaseStudyWater/CaseStudyWaterHeader";

describe("CaseStudyWaterHeader test", () => {
  it("Shows CaseStudyWaterHeader", () => {
    const domElement = document.createElement("p");
    ReactDOM.render(
      <MemoryRouter>
        <CaseStudyWaterHeader />
      </MemoryRouter>,
      domElement
    );
    expect(domElement.innerHTML).toMatchSnapshot();
  });
});
