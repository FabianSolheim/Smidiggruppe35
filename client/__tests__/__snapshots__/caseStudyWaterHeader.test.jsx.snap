// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CaseStudyWaterHeader test Shows CaseStudyWaterHeader 1`] = `"<div class=\\"TopHeader\\" style=\\"background-image: url(https://images.unsplash.com/photo-1596021876043-3ec2658c4477?ixlib=rb-1.2.1&amp;ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&amp;auto=format&amp;fit=crop&amp;w=2874&amp;q=80); background-repeat: no-repeat; background-size: cover; background-position: center; text-align: center;\\"><div class=\\"HeaderBox\\"><h1 class=\\"MuiTypography-root MuiTypography-h1 BoxHeadline css-o2w69a-MuiTypography-root\\" style=\\"color: black;\\">Water</h1><div class=\\"BoxText\\" style=\\"color: black;\\"><p><strong><em>785 million</em></strong> people in the world live <strong><em>without</em></strong><br> clean water. <br></p></div></div></div><div style=\\"text-align: center; margin-top: -15px;\\"><a class=\\" css-1a6d3yr-MuiButton-root\\" component=\\"button\\" focusvisibleclassname=\\"\\" style=\\"padding: 12px 22px; text-decoration: none; font-weight: 500; border-radius: 0; text-transform: none;\\" classes=\\"[object Object]\\" href=\\"/booking\\">Book a Consultation</a></div>"`;
